nvflare/fuel/hci/client/api.py

```diff
@@ -167,19 +170,29 @@ def _load_client_cmds(self, cmd_modules):
     def register_command(self, cmd_entry):
         self.all_cmds.append(cmd_entry.name)
 
-    def session_monitor(self, session_callback, interval=5):
-        _session_monitor_thread = threading.Thread(
+    def start_session_monitor(self, session_callback, interval=5):
+        if isinstance(self.sess_monitor_thread, threading.Thread) and self.sess_monitor_thread.is_alive():
```

Comment:
Can this check be simplified as 
`if self.sess_monitor_thread and self.sess_monitor_thread.is_alive()`

Reply:
Sure